chassis_hardware_node:
  ros__parameters:
    serial_port: "/dev/ttyCH341USB0"
    baud_rate: 115200
    base_frame_id: "base_link"
    odom_frame_id: "odom"
